content/influxdb/v2.0/reference/rc0-upgrade-guide.md

```diff
@@ -0,0 +1,294 @@
++-----+------------------------+----+---------------+---------+-------------+
+|                                                      TOTAL  |      1      |
++-----+------------------------+----+---------------+---------+-------------+
+​
+BUCKETS    +add | -remove | unchanged
++-----+------------------------+----+---------------+------------------+-------------+
+| +/- |     METADATA NAME      | ID | RESOURCE NAME | RETENTION PERIOD | DESCRIPTION |
++-----+------------------------+----+---------------+------------------+-------------+
+| +   | fasting-taussig-c9c007 |    | apps          | 0s               |             |
++-----+------------------------+----+---------------+------------------+-------------+
++-----+------------------------+----+---------------+------------------+-------------+
+| +   | great-davinci-c9c005   |    | new_telegraf  | 0s               |             |
++-----+------------------------+----+---------------+------------------+-------------+
++-----+------------------------+----+---------------+------------------+-------------+
+| +   | stubborn-hugle-c9c003  |    | telegraf      | 719h59m59s       |             |
++-----+------------------------+----+---------------+------------------+-------------+
+|                                                          TOTAL       |      3      |
++-----+------------------------+----+---------------+------------------+-------------+
+```
+
+{{% note %}}
+We are using config profiles here to export all the resources from the old instance and applying them to your new instance.
+The only things that will not be copied over are scraper configurations.
+You will need to manually reconfigure those.
+{{% /note %}}
+
+Now you have all the resources from your old instance stored in your new instance.
+Sign in to your new instance (by default http://localhost:8086) and take a look.
+You will see dashboards, but your old data has yet been migrated.
+
+## 8. Set up integrations to point to new instance
+
+Re-enable any integrations you disabled in step 2.
+Telegraf, client libraries, custom applications,
+or third-party data sinks will need to be set up again using new tokens and credentials.
+
+## 9. Load your historical data into new instance
+
+Use the CLI to export and then re-import your data using the command below.
+(For the range, pick a time before your bucket's retention period, or something a really long time ago if you have an unlimited retention period.)
+
+```sh
+influx query -c influx_old \
+  'from(bucket: "my-bucket") |> range(start: -3y)' --raw > my-bucket.csv
+```
+
+Then write to the new bucket:
+
+```sh
+influx write -c default --format csv -b my-bucket -f my-bucket.csv
+```
+
+Repeat that process for each bucket.
+
+## Verify InfluxDB resources, data, and integrations
+
+To verify that the latest version of InfluxDB is running with all your resources, data, and integrations configured,
+Double check and make sure everything is there and it is working as expected.
```

Comment:
Confused here-- are these two lines supposed to be one sentence? Also double-check needs a hyphen.